go/consensus/cometbft/apps/keymanager/churp/txs.go

```diff
@@ -417,6 +425,39 @@ func (ext *churpExt) computeNextHandoff(ctx *tmapi.Context) (beacon.EpochTime, e
 	return epoch + 1, nil
 }
 
+func addStakeClaim(ctx *tmapi.Context, entityID signature.PublicKey, runtimeID common.Namespace, churpID uint8) error {
+	regState := registryState.NewMutableState(ctx.State())
+
+	regParams, err := regState.ConsensusParameters(ctx)
+	if err != nil {
+		return err
+	}
+	if regParams.DebugBypassStake {
```

Comment:
As you mentioned, this could probably be moved to staking parameters. Since it is a debug parameter it is not breaking as no debug parameters can be set in production networks. But we can refactor later in a separate PR.